pages/docs/end-user-guide/nominating.mdx

```diff
@@ -0,0 +1,129 @@
+
+<img src="/img/nominating/nominate-01.png" width="100%" height="100%" />
+
+
+#### 2. Select Wallet & Stake Amount
+You will need select the wallet you wish to nominate from, also the amount. On the amount always leave some AVL for fees, so only nominate 95%-98% of your tokens. Levaing 2%-5% for fees on nominating or any other tranactions you may want to perform.
+Also select where you would like your rewards to acrue to. You have the following options
+
+- **Stash Account (Increase the amount at stake):** This refers to a scenario where the user increases the amount of AVL (Avail token) at stake with their rewards in their stash account. This is like auto-compunding, your rewards are added to your stake and will increase your overall stake for the next era.
+
+- **Stash Account (Do not increase the amount at stake):** In contrast to the previous point, here users choose not to increase the amount of AVL at stake in their stash account. Yur rewards will be sent to this account but ot staked.
+
+- **Controller Account:** The controller account is separate from the stash account and is used to manage the nomination process. This is legacy and will be discontinued going forward.
+
+- **Specified Payment Account:** This refers to the account designated by the user to receive staking rewards. When nominating, users can specify a payment account where their rewards will be deposited. This account can be different from both the stash and controller accounts and allows for flexibility in managing staking rewards.
+
+<img src="/img/nominating/nominate-02.png" width="100%" height="100%" />
+
+Once you have selected you may click on `+Next` button.
+
+#### 3. Select Validators to Nominate
+You can now select up to 16 validators to nominate. It is important to note your stake will support one, or several or all validators. At time of election the Avail concensus will select the best combination. 
+More information about selection can be found on the [<ins>NPoS Page</ins>](https://docs.availproject.org/docs/learn-about-avail/consensus/npos). Once you have selected you can click on `Bond & Nominate`.
+
+<img src="/img/nominating/nominate-03.png" width="100%" height="100%" />
+
+
+#### 4. Sign & Submit Your Selection
+On the final steps you must enter your password and confirm by clicking on `Sign and Submit`.
+<img src="/img/nominating/nominate-04.png" width="100%" height="100%" />
+
+
+You will now see your stake in `Waiting`, your stake will become active if you selecd validators that will be elected when the next era starts.
+<img src="/img/nominating/nominate-05.png" width="100%" height="100%" />
+
+
+### How to Join a Nomination pool
+
+#### 1. Navigate to Nomination Pools
+To join a nomination pools navigate to the [<ins>Pools Page</ins>](https://goldberg.avail.tools/#/staking/pools). Select a pool to join where the existing members are less than the max members per pool. Click on `+Join` to enter a pool.
```

Comment:
Same question ser. Do we have a hard number on the max number of members in a pool?

Reply:
At this moment the goldberg is 200 per pool. I however have recommended the team to increase as it is extremely limiting. Other substrates that use this do not cap at 200

This is a setting that can be changed with governance.